core/src/cluster_slots_service.rs

```diff
@@ -100,12 +108,15 @@ impl ClusterSlotsService {
             lowest_slot_elapsed.stop();
             let mut process_cluster_slots_updates_elapsed =
                 Measure::start("process_cluster_slots_updates_elapsed");
-            if let Some(slots) = slots {
-                Self::process_cluster_slots_updates(
-                    slots,
-                    &cluster_slots_update_receiver,
-                    &cluster_info,
-                );
+            // only staked nodes push EpochSlots into CRDS
```

Comment:
would be better to include the "why" here

Reply:
addressed in [df0ffb9](https://github.com/anza-xyz/agave/pull/5141/commits/df0ffb94e900d948431c764fe1f6cbc7ff53f63f)